tests/behat/features/capabilities/search/search-content.feature

```diff
@@ -35,7 +46,20 @@ Feature: Search
     And I should see "Event one" in the "Main content"
     And I should see "Topic one"
     And I should not see "Event second"
-    # Scenario: Successfully filter search results
+
+    # Test the pager.
+    When I am on "search/content"
+    And I fill in the following:
+      | search_input | Shenanigans |
+    And I press "Search"
+    And I click the xth "0" element with the css ".pager-nav .pager__item--next"
```

Comment:
What would be helpful for the future is to make a custom step definition, something like 
_"and I click the pager to go to the next page"_ or something which makes it more readable,